bitvision/bitvision.py

```diff
@@ -15,126 +15,160 @@
 CONFIG = configparser.ConfigParser()
 CONFIG.read("config.ini")
 
-
 #########
 # HELPERS
 #########
 
+# BitVision Command Line Trading Platform
+# Aaron Lichtman and Jon Shobrook
 
-def login():
+import os
+import sys
+import urwid
+
+# Place Your Own Trade is not shown until Trading Deactivated.
+choices = ["Activate/Deactivate Automated Trading", "Deposit/Withdraw Money", "Place Your Own Trade"]
+
+
+def menu(title, choices):
+	"""
+    Make a menu with choices displayed.
     """
+	body = [urwid.Text(title), urwid.Divider()]
+	for c in choices:
+		button = urwid.Button(c)
+		urwid.connect_signal(button, 'click', item_chosen, c)
+		body.append(urwid.AttrMap(button, None, focus_map='reversed'))
+	return urwid.ListBox(urwid.SimpleFocusListWalker(body))
```

Comment:
Cool idea making this a ListBox, but I think a simple Text widget would be a better fit for this (like the menu bar used in rebound). Here's an example:

```python
menu = urwid.Text([
    u'\n',
    ("menu", u" ENTER "), ("light gray", u" Automated Trading: ON "),
    ("menu", u" D "), ("light gray", u" Deposit Money "),
    ("menu", u" W "), ("light gray", u" Withdraw Money "),
    ("menu", u" P "), ("light gray", u" Place a Trade "),
    ("menu", u" Q "), ("light gray", u" Quit")
])
```

The user would press those keys to perform their respective actions. And we'd place this widget in the `footer` of a wrapping `Frame` widget.

Reply:
What if we do a status widget right above the options widget? Or figure out how to swap the text every time the button is pressed?